script/test_2.py

```diff
@@ -1,17 +1,32 @@
+import argparse
+import time
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
   
 
 def main():
-  driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--driver', help='entry driver')
+  args = parser.parse_args() 
+  driver_choice = args.driver  
+  if driver_choice == 'chrome':
+    driver = webdriver.Chrome(executable_path="bin/chromedriver")
+  else:
+    driver = webdriver.Firefox(executable_path="bin/geckodriver.exe")
   driver.get("https://en.wikipedia.org/wiki/Main_Page")
   search_input = driver.find_element_by_id("searchInput")
   search_input.clear()
   search_input.send_keys("Unit testing")
   search_input.submit()
+  time.sleep(3)
```

Comment:
Why sleep instead of WebDriverWait?

Reply:
When I run the script from --driver chrome, I get a WebDriverWait error :
```
$ python script/test_2.py --driver "chrome"

DevTools listening on ws://127.0.0.1:60079/devtools/browser/36b29f9e-238f-4919-a5ab-1a9cb3c51db0
Traceback (most recent call last):
  File "script/test_2.py", line 39, in <module>
    main()
  File "script/test_2.py", line 27, in main
    WebDriverWait(driver, 10).until(EC.url_changes(driver.current_url))
  File "C:\Users\annal\AppData\Local\Programs\Python\Python38-32\lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until     
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message:
```

Reply:
Try to wait for something else